CHANGELOG.md

```diff
@@ -4,17 +4,32 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
+#### [1.6.2](https://github.com/unbounce/laundry/compare/1.6.1...1.6.2)
```

Comment:
this file should be auto generated on release right, do we need to commit it manually

Reply:
The release step is manually triggered, I think.